scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java

```diff
@@ -233,12 +236,18 @@ public SchedulerDBManager(Configuration configuration, boolean drop) {
             if (offset >= 0) {
                 criteria.setFirstResult(offset);
             }
-            if (user != null) {
+            if (user != null && !user.isEmpty()) {
                 criteria.add(Restrictions.eq("owner", user));
             }
             if (!childJobs) {
                 criteria.add(Restrictions.isNull("parentId"));
             }
+            if (jobName != null && !jobName.isEmpty()) {
+                criteria.add(Restrictions.like("jobName", jobName, MatchMode.START).ignoreCase());
```

Comment:
not sure if ignoreCase is a good idea

Reply:
indeed, there is still this ignore case after your changes